spring-cloud-sleuth-stream/src/main/java/org/springframework/cloud/sleuth/stream/StreamSpanReporter.java

```diff
@@ -38,6 +38,13 @@
 @MessageEndpoint
 public class StreamSpanReporter implements SpanReporter {
 
+	/**
+	 * Bean name for the
+	 * {@link org.springframework.integration.scheduling.PollerMetadata
+	 * PollerMetadata}
+	 */
+	public static final String POLLER = "streamSpanReporterPoller";
```

Comment:
@dsyer etc.. is this a normal spring thing? I've not been doing that, but can. pro and con here is that the name is decoupled from the class so it will stay the same even if the class is renamed.